shuup/front/apps/simple_search/views.py

```diff
@@ -67,8 +67,8 @@ def get_search_product_ids(request, query, limit=150):
     if product_ids is None:
         entry_query = get_compiled_query(
             query, ['sku', 'translations__name', 'translations__description', 'translations__keywords'])
-        product_ids = Product.objects.filter(entry_query).distinct().values_list("pk", flat=True)[:limit]
-        cache.set(cache_key, product_ids, 60 * 5)
+        product_ids = list(Product.objects.filter(entry_query).distinct().values_list("pk", flat=True))
```

Comment:
I think you should change this to `product_ids = list(Product.objects.filter(entry_query).distinct().values_list("pk", flat=True))[:limit]` so that the cached and returned ids are the same